src/kafkerl_connector.erl

```diff
@@ -295,41 +306,41 @@ request_metadata(Pid, Brokers, TCPOpts, Retries, RetryInterval, Request) ->
                      -1 -> -1;
                      N  -> N - 1
                    end,
-      request_metadata(Pid, Brokers, TCPOpts, NewRetries, RetryInterval,
+      do_request_metadata(Pid, Brokers, TCPOpts, NewRetries, RetryInterval,
                        Request)
   end.
 
-request_metadata([], _TCPOpts, _Request) ->
+do_request_metadata([], _TCPOpts, _Request) ->
   {error, all_down};
-request_metadata([{Host, Port} = _Broker | T] = _Brokers, TCPOpts, Request) ->
+do_request_metadata([{Host, Port} = _Broker | T] = _Brokers, TCPOpts, Request) ->
```

Comment:
Line 315 is too long: do_request_metadata([{Host, Port} = _Broker | T] = _Brokers, TCPOpts, Request) ->.